src/main/java/hudson/plugins/performance/reports/UriReport.java

```diff
@@ -190,8 +194,8 @@ public long getAverage() {
     }
 
     private long getDurationAt(double percentage) {
-        if (percentage < 0 || percentage > 1) {
-            throw new IllegalArgumentException("Argument 'percentage' must be a value between 0 and 1 (inclusive)");
+        if (percentage < ZERO_PERCENT || percentage > ONE_HUNDRED_PERCENT) {
```

Comment:
Here no errors, because i changed all input params to following double constants:
1) ZERO_PERCENT=0
2) NINETY_PERCENT=90
3) FIFTY_PERCENT=50
4) ONE_HUNDRED_PERCENT=100.
This approach will allow to add other percentiles